apps/mark-scan/frontend/src/pages/poll_worker_screen.tsx

```diff
@@ -566,10 +566,18 @@ export function PollWorkerScreen({
 
   return (
     <Screen>
-      {!isLiveMode && <TestMode />}
       <Main padded>
         <div>
-          <H2 as="h1">Poll Worker Menu</H2>
+          <div
+            style={{
+              display: 'flex',
+              justifyContent: 'space-between',
+              alignItems: 'start',
```

Comment:
Wondered if it might look better with "center" when it is on the same row as the title? I'm sure you already tested, just sharing the thought

Reply:
Yeah, I tried but it was weirdly offset and I didn't want to have to manually tweak the positioning